app/contributing/plugin-docs.md

```diff
@@ -44,6 +43,42 @@ should be a square-format PNG file, 120x120 pixels in size.
 
     Add the icon file to `/app/_assets/images/icons/hub/`. 
 
+### Third-party or partner plugins
+
+1. Set up or find your publisher directory in the [docs repository](https://github.com/Kong/docs.konghq.com/tree/main/app/_hub/).
+
+   If your company has not contributed a plugin to Kong before, 
+   create a publisher directory at`_app/_hub/`, such as
+  `_app/_hub/company-name`. 
+
+    See other Kong Hub listings for examples of publisher names.
+
+1. Create a subdirectory for your plugin within your publisher directory.
+For example, `_app/_hub/kong-inc/your-plugin`.
+
+1. Copy the [`/app/_hub/_init/my-extension/_index.md`](https://github.com/Kong/docs.konghq.com/tree/main/app/_hub/_init/my-extension)
+ file into your plugin's subdirectory.
+
+    **Do not** use the `versions.yml` file. 
```

Comment:
```suggestion
    **Do not** use the `versions.yml` file. 
```

Don't do anything with it? Why is it here? Is it worth explaining?

Reply:
The explanation is the following line, ie "Kong does not maintain multiple versions of third-party plugin docs."

I put this here because we've had multiple third-party contributors try to use this file now - and I can understand why. Every kong-inc plugin has one, and there's a `versions.yaml` in the template folder for the plugin (https://github.com/Kong/docs.konghq.com/tree/main/app/_hub/_init/my-extension)